src/Context.php

```diff
@@ -210,4 +212,13 @@ public function offsetUnset(mixed $offset): void
     {
         throw new \LogicException('Context is immutable');
     }
+
+    public static function fromPath(string $path): self
+    {
+        $dotenv = new Dotenv();
+        $dotenv->loadEnv($path);
+        unset($_ENV['SYMFONY_DOTENV_VARS']);
```

Comment:
why ?

Reply:
it's added by the Symfony's DotEnv component and is only used internally, no need  to keep it in our case.